README.md

```diff
@@ -92,137 +54,110 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Create animal table**
+- **Populate table with animal data**
+- **Query data**
+- **Update data**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Coming soon...
+> Add a link to your deployed project.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+- Coming soon....
 
-<!-- GETTING STARTED -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
-
 ```sh
  gem install rails
 ```
- -->
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
 ```sh
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone https://github.com/shaaibu7/Vet-Clinic-Database
 ```
---->
+
+-
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
 ```sh
-  cd my-project
+  cd vet_clinic
   gem install
 ```
---->
+
+-
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
   rails server
 ```
---->
+
+-
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
 ```sh
   bin/rails test test/models/article_test.rb
 ```
---->
+
+-
 
 ### Deployment
 
 You can deploy this project using:
 
-<!--
-Example:
-
 ```sh
 
 ```
- -->
 
```

Comment:
- [ ] Your Getting Started section is not descriptive because it contains commands which are not related to this project 😨 Kindly add the following commands in the Getting Started Section ⬇️

## 💻 Getting Started <a name="getting-started"></a>
To get a local copy up and running, follow these steps.
### Prerequisites
```
PostgreSQL
Visual Studio Code
```
### Setup
```
  cd my-folder
  git clone https://github.com/shaaibu7/Vet-Clinic-Database
  cd Vet-Clinic-Database
```
### Usage
```
This repository includes files with plain SQL that can be used to recreate a database:

- Use [schema.sql](./schema.sql) to create all tables.
- Use [data.sql](./data.sql) to populate tables with sample data.
- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database.
```